db/CouriersDB.go

```diff
@@ -0,0 +1,62 @@
+package db
+
+import (
+	"fmt"
+	"strconv"
+)
+
+type Courier struct {
+	IdCourier        uint16 `json:"id_courier"`
```

Comment:
usually the naming convention for db is not repeating the name of the table in field name. Just `id` or `name` is enough.

Reply:
Primary key becomes just `id`, secondary keys continue having the prefix, because for them it isn't the name of current table, so it doesn't violates the rule. Though usually the convention is to call it `order_id` rather than `id_order`.